ssn/integrated/ssn.ttl

```diff
@@ -46,14 +46,66 @@ ssn: a owl:Ontology ;
 ssn:Property a owl:Class ;
   rdfs:label "Property"@en ;
   skos:definition """A Quality of a FeatureOfInterest. An aspect of an entity that is intrinsic to and cannot exist without the entity."""@en ;
-#  rdfs:subClassOf [ owl:onProperty sosa:isPropertyOf ; owl:someValuesFrom sosa:FeatureOfInterest ] ; # TODO: need resolution, is it ssn:isPropertyOf, or sosa:isPropertyOf ?
+#  rdfs:subClassOf [ owl:onProperty sosa:isPropertyOf ; owl:someValuesFrom sosa:FeatureOfInterest ] ; # TODO: need resolution, see https://www.w3.org/2015/spatial/wiki/Link_between_FeatureOfInterest_and_xxxProperty
   rdfs:isDefinedBy ssn: .
 
+  ssn:hasProperty a owl:ObjectProperty ; # TODO: need resolution, see https://www.w3.org/2015/spatial/wiki/Link_between_FeatureOfInterest_and_xxxProperty
+    rdfs:label "has property"@en ;
+    skos:definition """Relation between a FeatureOfInterest and a Property of that feature."""@en ;
+    owl:inverseOf ssn:isPropertyOf ;
+    rdfs:isDefinedBy ssn: .
+
+  ssn:isPropertyOf a owl:ObjectProperty ; # TODO: need resolution, see https://www.w3.org/2015/spatial/wiki/Link_between_FeatureOfInterest_and_xxxProperty
+    rdfs:label "is property of"@en ;
+    skos:definition """Relation between a FeatureOfInterest and a Property (a Quality observable by a sensor) of that feature."""@en ;
+    owl:inverseOf ssn:hasProperty ;
+    rdfs:isDefinedBy ssn: .
```

Comment:
added very simple ssn:hasProperty and ssn:isPropertyOf, as these properties will probably not be in SOSA